modules/account-lib/src/coin/dot/proxyBuilder.ts

```diff
@@ -0,0 +1,145 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { decode, methods } from '@substrate/txwrapper-polkadot';
+import { InvalidTransactionError } from '../baseCoin/errors';
+import { TransactionBuilder } from './transactionBuilder';
+import { Transaction } from './transaction';
+import { UnsignedTransaction } from '@substrate/txwrapper-core';
+import { TransactionType } from '../baseCoin';
+import { MethodNames, ProxyArgs, proxyType } from './iface';
+import { ProxyTransactionSchema } from './txnSchema';
+import utils from './utils';
+
+export class ProxyBuilder extends TransactionBuilder {
```

Comment:
Is this the wrapper transaction for the AddProxy method?

Reply:
This is the wrapper transaction for any transaction we want to execute on the network through a proxy